cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -191,3 +191,303 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
 
     # Revert back to original config
     reconfigure_vm(small_vm, orig_config)
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.parametrize('change_type', ['sockets', 'memory'])
+def test_vm_reconfig_add_remove_hw_hot_vmware(change_type):
+    """
+    Polarion:
+        assignee: nansari
+        initialEstimate: 1/6h
+        caseimportance: high
+        caselevel: integration
+        caseposneg: positive
+        caseautomation: Non Automated
```

Comment:
should be `caseautomation: notautomated` here and in all other places
see https://github.com/ManageIQ/integration_tests/blob/master/conf/polarion_tools.yaml#L88

Reply:
As per our new updates. we are avoiding this `key`. `casautomation` taken care by `tier` marker.